src/poliastro/core/iod.py

```diff
@@ -9,7 +9,7 @@
 
 
 @jit
-def vallado(k, r0, r, tof, short, numiter, rtol):
+def vallado(k, r0, r, tof, M, direct, short, numiter, rtol):
```

Comment:
Can we _not_ call this `M`? At first I thought this was supposed to be the true anomaly. I know, I know: this is the name we used for `izzo`, but in retrospect I now think it was bad :joy: No need to fix it there though, just use `num_rev` here.

Reply:
Regarding the confusion between `direct` and `short` that @jorgepiloto already mentioned, the MATLAB code makes a distinction between:

* "Direction of motion": long period vs short period
* "Direction of flight": direct vs retrograde

However, I don't see that distinction mentioned in the book:

![Screenshot from 2020-04-24 13-18-53](https://user-images.githubusercontent.com/316517/80207274-2a07e700-862e-11ea-80d9-ab092e1fd1a1.png)

Look at these disturbing comments in the MATLAB code:

```matlab
if ( df == 'r' )  %dm == 'l'
...
if (df == 'd')  % dm == 's'
```

Does that mean that they are the same?

To simplify things, let's consider only **prograde orbits**, and maintain the `short` parameter.

Reply:
Now that I think of it, once you've decided it's short of long there's **only one way** to go.  Like in Fig 7 you showed,  once you've chosen short that's it, no more "direct/retrograde". I think they're synonymous don't you agree?

Reply:
Yeah, let's consider them synonymous!